docs/install-1.6.md

```diff
@@ -27,14 +27,14 @@ KUBE_ENABLE_CLUSTER_DNS=true hack/local-up-cluster.sh -O
 
 ## Helm
 
-You *must* use [Helm](http://helm.sh/) v2 or newer in the installation steps
+You *must* use [Helm](http://helm.sh/) v2.5.0 or newer in the installation steps
 below.
 
-If you already have Helm v2 or newer, execute `helm init` (if you haven't
+If you already have Helm v2.5.0 or newer, execute `helm init` (if you haven't
```

Comment:
Repeating the version 3 times in a single section looks a bit redundant to me, I would leave if only for the first mention and omit the version completely in all other places

Reply:
@nilebox I've removed redundancy